packages/valory/connections/ledger/ledger_dispatcher.py

```diff
@@ -240,23 +259,41 @@ def get_transaction_receipt(
             and attempts < retry_attempts
             and self.connection_state.get() == ConnectionStates.connected
         ):
-            transaction_receipt = api.get_transaction_receipt(
-                message.transaction_digest.body
-            )
+            try:
+                transaction_receipt = api.get_transaction_receipt(
+                    message.transaction_digest.body,
+                    raise_on_try=True,
+                )
+            except Exception as e:  # pylint: disable=broad-except  # pragma: nocover
+                return self.get_error_message(e, api, message, dialogue)
+
             if transaction_receipt is not None:
                 is_settled = api.is_transaction_settled(transaction_receipt)
             attempts += 1
             time.sleep(retry_timeout * attempts)
```

Comment:
This is the reason that the repricing test fails. Previously, it would just sleep and loop until settling, if attempts were not exceeded. Now, it will return an error message and loop. Therefore, we need to either:
1. Update the test in order to sleep before `get_transaction_receipt` to avoid getting error messages. 
2. Account for a number of error messages.

Getting an error message before the transaction settles is normal.

Reply:
Fixed in fcf5341fddd15b2ea92d49c44448b7a05352801a.